src/main/java/com/alkemy/ong/mail/EmailService.java

```diff
@@ -0,0 +1,42 @@
+package com.alkemy.ong.mail;
+
+import com.sendgrid.Method;
+import com.sendgrid.Request;
+import com.sendgrid.Response;
+import com.sendgrid.SendGrid;
+import com.sendgrid.helpers.mail.Mail;
+import com.sendgrid.helpers.mail.objects.Content;
+import com.sendgrid.helpers.mail.objects.Email;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+import java.io.IOException;
+
+@Service
+public class EmailService {
+
+    @Value("${app.sendgrid.key}")
+    private String apikey;
+
+    @Value("${from.email}")
```

Comment:
Could you plz add the application.properties for this values? (without revealing sensitive information off course)